tools/data/USD.patch

```diff
@@ -0,0 +1,88 @@
+diff --git a/pxr/imaging/hioOpenVDB/CMakeLists.txt b/pxr/imaging/hioOpenVDB/CMakeLists.txt
```

Comment:
So my only question here is about this patch file and why its necessary.  Could we merge these changes or change hdRPR to allow?

Reply:
There is a bug in USD with hioOpenVDB project. It was created without public interface, which produces build error in HdRPR. This patch fixes it. We also can provide this fix to USD project, create PR.

Reply:
Please do.

Reply:
Discussed with @hshakula, he will do such PR to USD project.

Reply:
https://github.com/PixarAnimationStudios/USD/pull/1574
https://github.com/PixarAnimationStudios/USD/pull/1575